desk/app/focus.hoon

```diff
@@ -43,62 +59,284 @@
 ::
 ++  on-init
   ^-  (quip card _this)
-  :-  ~[(~(connect pass:io /connect) [[~ /[dap.bowl]] dap.bowl])]
-  this(state [%0 [~m5 8 1 ~s30 8] 1 [now.bowl now.bowl] [%enter %focus | %fresh |] |])
+  :-
+  :~  (~(connect pass:io /connect) [[~ /[dap.bowl]] dap.bowl])
+  ==
+  %=  this
+    state  :*  %1
+               [~m5 8 1 ~s30 8]
+               1
+               [now.bowl now.bowl]
+               [~s0 ~s0]
+               [%enter %focus | %fresh]
+               [| [our.bowl *@da] [our.bowl *@da] [our.bowl *@da] [our.bowl *@da]]
+               |
+           ==
+  ==
 ::
 ++  on-save  !>(state)
 ::
 ++  on-load
   |=  saved=vase
   ^-  (quip card _this)
   ~&  ~(key by pages)
-  =/  ole  !<(versioned-state saved)
-  `this(state ole)
+  =/  last-save  !<(versioned-state saved)
+  ::  bunting state-1 should give a clean slate
+  ?-  -.last-save
+    %1  `this(state last-save)
+    %0  `this(state *state-1)
```

Comment:
bunting state-1 was a mistake.
the `*?` is `%.y`, that causes problems with goals and making things public.